packages/terra-search-field/src/terra-dev-site/test/search-field/DefaultSearchField.test.jsx

```diff
@@ -2,4 +2,4 @@ import React from 'react';
 import SearchField from '../../../SearchField';
 
 export default () =>
-  <SearchField id="searchfield" />;
+  <SearchField id="searchfield" inputAttributes={{ 'aria-label': 'Search Field' }} />;
```

Comment:
This removes the need for accessibility in our test examples, however, should we be indicating to consumers that an ariaLabel should be included?

Reply:
Currently in the description of inputAttributes on terra-dev-site it reads

> Custom input attributes to apply to the input field such as aria-label.

I am open to better conveying that the consumer needs to define aria-label but unsure of what would be the better way to do so.

Tangential note:
Would there be value in having the default for inputAttributes including a defined Aria-Label?

Reply:
I'm concerned users may not know to include  aria-labels for this, even if we document it. Instead, I'd like to move forward with providing defaults for aria-labels for the input and the button. If the user passes in `inputAttributes` with `aria-label` on it, we could use that instead of the default we provide. This way, users get a form input / button combo that has i18n labels by default, but still have the option to pass in their own input label if they want.